src/components/Summary/AlterationSummary.vue

```diff
@@ -140,10 +140,13 @@
           </v-col>
 
           <v-col cols="9" class="inner-col-2">
-            <p class="info-text">Select the date and time of alteration of your business. You may select a date and
-              time up to 10 days in the future (note: there is an <strong>additional fee of $100.00</strong> to enter
-              an alteration date and time in the future). Unless a business has special requirements, most businesses
-              select an immediate Alteration Date and Time.
+            <p
+              id="effective-date-time-instructions"
+              class="info-text">
+              Select the date and time of alteration of your business. You may select a date and time up to 10 days in
+              the future (note: there is an <strong>additional fee of ${{futureEffectiveFees.toFixed(2)}}</strong> to
```

Comment:
Could this number ever be returned beyond 2 decimals?

Reply:
No, it is returned with 1 decimal actually. It's just to make it uniform. And in the future it may come different from payment-api.

Reply:
This is OK with me. There are other places in the code that use `toFixed(2)` for amounts.